src/utils.js

```diff
@@ -61,7 +61,7 @@ function clampValue(number, min, max) {
 }
 
 function fallbackColor(color, fallbackColor) {
-  return (isValidHexColor(color) && `#${color}`) || fallbackColor;
+  return (isValidHexColor(color) && `#${color}`) || (color.includes(',') && color.split(',')) || fallbackColor;
```

Comment:
This is the (new) algorithm I use to detect gradients. It could use some work though to check whether the user provides valid hex colors and valid angles...

Reply:
Resolved as now hex color are verified. Angles are not, however I didn't see that as a big issue; if a user enters 360˚ will appear to be at 0˚, and it does not cause svg errors. (as far as I know...)